main.tf

```diff
@@ -147,7 +147,9 @@ resource "aws_s3_bucket" "origin" {
 }
 
 module "logs" {
-  source                   = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.14.0"
+  # TODO: merge https://github.com/cloudposse/terraform-aws-s3-log-storage/pull/42 before submitting, update tag
```

Comment:
This is a dependent PR that should be merged before this one

Reply:
It's been merged 👍